### ISSUE TITLE
Android Application Name Change to "Assurance" After Installing "@adobe/react-native-aepassurance" Library

### ISSUE THREAD

**Issue by ayushmahajan12** [adobe/aepsdk-react-native#378]
### Prerequisites

- [X] This is not a Security Disclosure, otherwise please follow the guidelines in [Security Policy](https://github.com/adobe/aepsdk-react-native/security/policy).
- [X] I have searched in this repository's issues to see if it has already been reported.
- [X] I have updated to the latest released version of the SDK and the issue still persists.

### Bug summary

The Android application name updates unexpectedly after running the app in both Development and Production modes following the installation of the @adobe/react-native-aepassurance library. This issue has been observed across different versions of the library.

### Environment

Operating System: Android 12, 13, and 14
React Native Version: 0.72.4 and 0.74.2
@adobe/react-native-aepassurance Version: 6.0.1 and 6.0.2

### Steps to reproduce

1. Create a new React Native project.
2. Run the Android app on your device.
3. Install the @adobe/react-native-aepassurance library.
4. Run the Android app on your device again.
5. Observe that the app name has changed i.e. "**Assurance**" instead of "**TestPlugin**"

### Current behavior

The application name updates to a different name i.e. "Assurance" after the library is installed and the app is run on an Android device.

### Expected behavior

The application name should remain consistent and not change after installing the @adobe/react-native-aepassurance library.

### Anything else?

The issue is consistently reproducible across multiple Android versions (12, 13, and 14).

![image](https://github.com/user-attachments/assets/44ff89e5-eda3-4042-90f5-daf02d2b0d30)
<img width="1512" alt="Screenshot 2024-08-15 at 5 34 12 PM" src="https://github.com/user-attachments/assets/a773f75c-b4f5-42ac-9c8b-ff3b998121ec">

**Comment by yangyansong-adbe** [comment]
Hi @ayushmahajan12,

I tried to reproduce the issue locally by following the steps you outlined, but the app name remained unchanged.
By default, the name of the Android app is configured within the app module's src->main->res->values->strings.xml file, such as: 
```xml
<resources>
    <string name="app_name">AwesomeProject</string>
</resources>
```
Installing our react native libraries will not update or override this file. You can also take a look at the name of the sample app hosted in this repository from [here](https://github.com/adobe/aepsdk-react-native/blob/main/apps/AEPSampleApp/android/app/src/main/res/values/strings.xml#L2).

Please let us know if you discover anything new.

**Comment by ayushmahajan12** [comment]
@yangyansong-adbe Could you please suggest why this is happening? Steps is too easy you can also test at your end.

**Comment by yangyansong-adbe** [comment]
Hi @ayushmahajan12 
I am now able to reproduce the issue and analyze it. I will provide more updates later.

**Comment by ayushmahajan12** [comment]
@yangyansong-adbe , if possible, could you please provide a solution by today? This issue is affecting my release.

Thanks in advance.

**Comment by yangyansong-adbe** [comment]
@ayushmahajan12 
Yes, we're currently working on the issue analysis. I'm not sure if we'll be able to release a new build today. An alternative solution would be to remove the Assurance library from your project to unblock the app release. Does that make sense?

**Comment by yangyansong-adbe** [comment]
@ayushmahajan12 
The root cause of this issue originates from our Android libraries that React Native libraries depend on. We are actively working on a quick solution and will release it as soon as we can. Could you kindly share all the Adobe libraries installed in your React Native app?  The problem occurs in several libraries. We will let you know as soon as all the impacted libraries are released.

**Comment by yangyansong-adbe** [comment]
@ayushmahajan12 
The problem has been addressed in the Android libraries. For additional information, please refer to the [release notes](https://developer.adobe.com/client-sdks/home/release-notes/).
I tested it on the React Native apps, and the `app name` remained unchanged after installing `@adobe/react-native-aepassurance`.
I will close this issue. If you still see the same problem occur, please feel free to reopen it or create a new issue.
